components/Header/MobileMenu.tsx

```diff
@@ -1,30 +1,30 @@
-import { LottieAnimation } from "components/LottieAnimation/LottieAnimation";
 import { grey200, grey500, links, socialLinks, white } from "constant";
-import { useInView } from "framer-motion";
+import dynamic from "next/dynamic";
 import NextLink from "next/link";
 import SmUpRightArrow from "public/assets/sm-up-right-arrow.svg";
-import { useEffect, useRef, useState } from "react";
+import { useEffect, useState } from "react";
 import styled, { CSSProperties } from "styled-components";
 import { isExternalLink } from "utils";
 
+const LottieAnimation = dynamic(() => import("components/LottieAnimation/LottieAnimation"));
+
 interface Props {
   show: boolean;
   hide: () => void;
   isLightTheme: boolean;
 }
 
 export function MobileMenu({ show, hide, isLightTheme }: Props) {
-  const ref = useRef<HTMLDivElement>(null);
-  const isInView = useInView(ref);
   const [animationData, setAnimationData] = useState<object>();
 
   useEffect(() => {
-    void import("public/assets/lottie/hero.json").then(setAnimationData);
-  }, []);
+    if (show) {
+      void import("public/assets/lottie/hero.json").then(setAnimationData);
```

Comment:
these lottie files are large and they also take a good chunk of time to parse, so we only want to load and parse them as we need them.